apps/cbportal/base/cloudbroker__cloudspace/methodclass/cloudbroker_cloudspace.py

```diff
@@ -136,6 +136,42 @@ def removeIP(self, cloudspaceId, ipaddress, **kwargs):
         """
         return True
 
+    @auth(['level1', 'level2', 'level3'])
+    @wrap_remote
```

Comment:
this wrap_remote decorator is not needed as we are not calling an APis from here